### PR TITLE
Navigation bar

### PR DESCRIPTION
**Pull request by ducaoga**
-made the navbar 
-only the title and home has routes to root.
-i accidentally made views and controllers which was supposedly for the next PR. 
-added gemfile for bootstrap
-added custom.scss

### CODE REVIEW

**Comment by ducaoga** [comment]
A lot of files were conflicted and some were deleted. so I had to recreate the files.

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+
+	def full_title(page_title ='')
+		base_title = "Ruby on Rails Tutorial Sample App"
+		if page_title.empty?
+			base_title
+		else
+			page_title + " | " + base_title
+		end
+	end
```
Comment (inahsanjuan) [inline app/helpers/application_helper.rb]: @ducaoga Use 2-spaced indentation not tabs.
Please update.

File: app/assets/stylesheets/custom.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+#logo{
+	float: left;
+	margin-right: 10px;
+	font-size: 1.7em;
```
Comment (inahsanjuan) [inline app/assets/stylesheets/custom.scss]: @ducaoga Update to 2-spaced indentation

File: app/views/layouts/_navbar.html.erb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      <%= link_to "E-Learning System", root_path, id: "logo" %>
+    </div>
+    <div class="container">
+      <ul class="nav navbar-nav navbar-right">
+        <li><%=  link_to "Home", root_path %></li>
+        <li><%=  link_to "Categories", "#" %></li>
+        <li><%=  link_to "Log In", "#" %></li>
+        <li><%=  link_to "Sign Up", "#" %></li>
+      </ul>
+    </div>
```
Comment (inahsanjuan) [inline app/views/layouts/_navbar.html.erb]: @ducaoga Update to 4-spaced indentation

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
+    <%= render 'layouts/navbar' %>
     <%= yield %>
   </body>
 </html>
```
Comment (inahsanjuan) [inline app/views/layouts/application.html.erb:15]: @ducaoga Update to 4-spaced indentaton

**Review by inahsanjuan — CHANGES_REQUESTED** [review]
OK
- Navbar and nav items are displayed

Comments
- Please follow correct coding standard:
https://docs.google.com/document/d/18OKPCS2CXkVU6Ih_spreTAMR3uECHudIBwvSZLdnk1U/edit
- Note: We are going to be using spaces, not tabs
**Example of tab indentation**
image
**Example of space indendation**
image
- Please research how to convert spaces to tabs

**Pull request merged**